exercises/high-scores/high_scores_test.rb

```diff
@@ -0,0 +1,55 @@
+require 'minitest/autorun'
+require_relative 'high_scores'
+
+# Common test data version: 1.0.0 2f612cc
+class HighScoresTest < Minitest::Test
+  def test_returns_the_highest_score
+    # skip
+    assert_equal 10, HighScores.new([4, 10, 7]).highest
+  end
+
```

Comment:
Should we have a test for returning all `scores` ? The unsorted list.

Reply:
Good question. Do we want `scores` to be public?

Reply:
Good question... I wanted to avoid the difference between private/public in this exercise. And even the `attr_reader :words`. The only way I see to accomplish that, is by making everything public expplicitly. :-) 
That's right, right? Then they can't use an instance variable, and chances are they're going for the `scores` method? And when they  already _know_ about `attr_reader` they will use that. 

Sound good?

Reply:
Ok. So yes to adding a spec for that, right?

Reply:
Yes to adding a spec 👍

Reply:
(You probably noticed, but this is now done :+1:)